tools.yaml

```diff
@@ -2177,6 +2177,13 @@
   categories:
   - opensource
   - author
+- name: NTIA Validator for CycloneDX
+  publisher: FOSSA
+  description: Ensure your CycloneDX SBOM meets NTIA requirements BEFORE you submit.
```

Comment:
Unsure of what "BEFORE you submit" means.   Relative to what process?  Borrowing from the "key features" section of the tool website, perhaps the description would better include the bullets from there: 
- Detailed validation feedback
- Dependency graph visualization and validation

Reply:
Thanks for the feedback. I've updated the description to explain a little better :)